DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/EditGroupName.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model EditGroupNameViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = $"{(errorHasOccurred ? "Error" : "")} Edit group name";

```

Comment:
Format is incorrect for the error case, it should be "Error: Edit group name" (with the colon)

Reply:
Ran the formatter

Reply:
Sorry, should have chosen my words more carefully. The string is wrong, it needs to result in "Error: Edit group name" when there is an error.

Reply:
Sorry, I updated the error message